### PR TITLE
Merging in to main from our branch

### PR DESCRIPTION
**Pull request by zackaad**
We corrected the failing test. :-D

### CODE REVIEW

**Review by emma7636 — APPROVED** [review]
Looking good :)

**Review by MagnusHJensen — APPROVED** [review]
LGTM

**Review by mathiasgredal — CHANGES_REQUESTED** [review]

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/Search/IsearchIndexedHTML.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package dk.sdu.se_f22.contentmodule.infrastructure.domain.Search;
+
+
+import dk.sdu.se_f22.sharedlibrary.models.Content;
+
+import java.util.List;
+
+public interface IsearchIndexedHTML {
```
[the remark this answers is not in the document]
Comment (zackaad) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/Search/IsearchIndexedHTML.java:8]: Hvorfor vil du have at interfacet arver IndexingModule? :)

[the remark this answers is not in the document]
Comment (mathiasgredal) [reply src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/Search/IsearchIndexedHTML.java:8]: Pga. dependency inversion, og fordi brand index og product index begge gør det samme.

File: src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/Search/SearchClass.java [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
-        if(searchResultHTMLIds.isEmpty()){
-            throw new NoResultsFoundException("No Search results found");
+        ArrayList<Integer> searchResultHTMLIds = CMSIndexModule.search(searchTokens);
+
+        if(!searchResultHTMLIds.isEmpty()){
+            return CMSModule.searchResults(searchResultHTMLIds);
         } else {
-            //ArrayList<Content> contents = new ArrayList<Content>(CMSModule.whatevertheirsearchmethodiscalled(searchResultHTMLIds));
-            return contents;
+            return null;
```
Comment (mathiasgredal) [inline src/main/java/dk/sdu/se_f22/contentmodule/infrastructure/domain/Search/SearchClass.java:24]: Kan det her være en tom liste i stedet?

**Review by mathiasgredal — CHANGES_REQUESTED** [review]

**Comment by MagnusHJensen** [comment]
Damn man @Autowinto 
I see you 👀

**Pull request merged**